From 9d66def1ed375a14a2a68a04233c83ea169cc687 Mon Sep 17 00:00:00 2001
From: Neena Busireddy <neenareddy.busireddy@nxp.com>
Date: Thu, 25 Jul 2019 17:24:43 +0000
Subject: [PATCH] workspace.bzl: Fix Eigen download issue

The following url does not work so use the alternate one.
http://mirror.tensorflow.org/bitbucket.org/eigen/eigen/get/a0d250e79c79.tar.gz

Signed-off-by: Neena Busireddy <neenareddy.busireddy@nxp.com>
---
 tensorflow/workspace.bzl | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorflow/workspace.bzl b/tensorflow/workspace.bzl
index e3ffdaf..61756f4 100755
--- a/tensorflow/workspace.bzl
+++ b/tensorflow/workspace.bzl
@@ -152,7 +152,6 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
         sha256 = "0dde8fb87f5dad2e409c9f4ea1bebc54e694cf4f3b633081b0d51a55c00f9c9f",
         strip_prefix = "eigen-eigen-a0d250e79c79",
         urls = [
-            "http://mirror.tensorflow.org/bitbucket.org/eigen/eigen/get/a0d250e79c79.tar.gz",
             "https://bitbucket.org/eigen/eigen/get/a0d250e79c79.tar.gz",
         ],
     )
-- 
2.7.4

